workspaces/ui-v2/src/styles/theme.ts

```diff
@@ -34,6 +36,16 @@ export const methodColorsDark: { [key: string]: string | undefined } = {
   DELETE: '#be5353',
 };
 
+export const jsonTypeColors = {
```

Comment:
🎉

Reply:
I'm not sure what the type representation looks like here, but I think we can set exhaustive type checking by specifying the type signature as `Record<JsonType, string>`

Reply:
It seems to be exhaustive so far, with TSC complaining when I didn't have `OBJECT` and `ARRAY` covered. But go nuts if I don't have to complete picture!

Reply:
This is more if we miss a variant of JsonType in this object, typescript will start yelling